components/eamxx/src/physics/ace/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+set(ACE_SRCS
+  eamxx_ace_process_interface.cpp
+  ace_grids_manager.cpp
+)
+
+# Initialize PyTorch found status to FALSE
+set(SCREAM_FOUND_PYTORCH FALSE CACHE BOOL "Whether PyTorch was found" FORCE)
+
+# Use a configurable path for PyTorch with a default value
+if(NOT DEFINED TORCH_CMAKE_PATH)
```

Comment:
I would rather this to be put in `cime_config/machines/cmake_macros/pm-gpu.cmake` and similar files. Then, in here, you should do
```cmake
if (NOT TORCH_CMAKE_PATH)
  message (FATAL_ERROR "bad boy")
endif()
```
Possibly, you could also add CMake checks that ensure the input path exists (and maybe that it is indeed a folder, or even that it is readable, but these may be ovrerkill), to avoid issues with typos.